pkg/build/node/node.go

```diff
@@ -365,15 +375,21 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 			}
 			// TODO(bentheelder): generate a friendlier name
 			pullName := fmt.Sprintf("%d.tar", i)
-			pullTo := fmt.Sprintf("%s/bits/images/%s", dir, pullName)
+			pullTo := fmt.Sprintf("%s/%s", imagesDir, pullName)
```

Comment:
we should use `path.Join` because this is on the host so the separator may not be `/`, whereas the paths "on the node" will be linux paths. (in practice this code should generally be fine, but we should be consistent anyhow)
```suggestion
			pullTo := path.Join(imagesDir, pullName)
```